src/cdk/v2/destinations/koala/procWorkflow.yaml

```diff
@@ -0,0 +1,65 @@
+bindings:
+  - name: EventType
+    path: ../../../../constants
+  - path: ../../bindings/jsontemplate
+  - name: defaultRequestConfig
+    path: ../../../../v0/util
+
+steps:
+  - name: validateInput
+    template: |
+      $.assert(.message.type, "message Type is not present. Aborting message");
+      $.assert(.message.type in {{$.EventType.([.IDENTIFY, .TRACK])}}, 
+        "message type " + .message.type + " is not supported");
+      $.assertConfig(.destination.Config.publicKey, "publicKey is not present. Aborting message");
+      $.context.email = .message.().({{{{$.getGenericPaths("email")}}}});
```

Comment:
@yashasvibajpai @koladilip Should we use `emailOnly`?

Reply:
Yes

Reply:
`email` in **GenericFieldMapping.json** is returning "traits.email", "context.traits.email", "properties.email", "context.externalId.0.id".
While `emailOnly` is returning "traits.email", "context.traits.email", "properties.email"